### ISSUE TITLE
Update slides and recording for HTML lesson

### ISSUE THREAD

**Issue by naomiquinones** [Techtonica/curriculum#2678]
### Description

The existing slides need updating.

The accompanying recording will also need to be updated.

### Location

[HTML](https://github.com/Techtonica/curriculum/blob/main/web/html.md) lesson

### Why Is This Important

Participants should be made aware of basic A11y and SEO concepts to keep in mind when creating properly structured HTML. Toward this end, new slides should be added to discuss those two foundational topics, semantic elements, and the difference between block and inline elements. Also, a few corrections and updates are needed in the code examples, such as properly pairing html tags, adding the lang attribute, using the correct hierarchical structure of headings, and emphasizing the difference between similarly-named elements.

### Acceptance Criteria

The updated slides will contain new sections on A11y, SEO, semantic elements, clarification of HTML document structure, and updates to existing code examples.
The video walkthrough will also discuss the new/updated material.
